docs/cloud-edge/modules/user-agent-filter.md

```diff
@@ -0,0 +1,130 @@
+# User Agent Filter
+
+---
+
+## Overview
+
+With the User Agent Filter module you can allow or deny traffic through regular expression-based rules that are matched against the incoming user-agent header, which contains information about the requesting browser or application, including its name, version, and operating system.
```

Comment:
This is a long sentence to lead with. Suggesting:

With the User Agent Filter module, you can allow or deny traffic through regular expression-based rules that are matched against the incoming `User-Agent` header. The `User-Agent` header contains information about the requesting browser or application, including its name, version, and operating system.